smarsy/parse.py

```diff
@@ -121,3 +121,62 @@ def login():
                                     get_user_credentials(),
                                     get_headers())
     return response
+
+
+def bs_safeget(soup, *args):
+    """
+        Utility function used to get a content string from a
+        Beautiful Soup object and tuple of selectors. Returns False
+        if no object is found for the given selector
+        """
+    for arg in args:
+        selectedElems = soup.select(arg)
+    if selectedElems is not None and len(selectedElems) > 0:
+        return selectedElems[0]
+    return False
+
+
+def create_parents_dict(parent_html) -> dict:
+    """
+    Function receive HTML and return parent dictionary with given keys
+    """
+    parent_img = bs_safeget(parent_html, '[valign=top]', 'img[src]')
+    # parent_dict = {}
+    # parent_list = []
+    # parents_keys = ['parent_img', 'parent_name', 'parent_surname',
+    #                 'parent_middlename', 'parent_type',
+    #                 'parent_birth_date']
+    if parent_img:
+        if hasattr(parent_img, 'src'):
```

Comment:
what does this if do? what is checked here? Isn't parent_img should be URL to image?

Reply:
No, parent_img is like this <img src='parent1.jpg'/> But we need only a part in src

Reply:
you didn't show anything

Reply:
`<img src="parent1.jpg"/>`